### ISSUE TITLE
Feature Request: Tray Icon

### ISSUE THREAD

**Issue by ghost** [gpodder/gpodder#195]
I keep gPodder running throughout the day and would love to have the ability to minimise it to the tray. I currently use a third-party tool to let me do this (RBTray) but it would be more ideal to have gPodder handle it natively.

Cheers!

**Comment by elelay** [comment]
Yes, the gtk tray icon extension works again! Click the icon to show/hide the main window.
Combine with the minimize on start extension...

**Comment by jalalsfs** [comment]
I see, as and extension, it's working and showing the tray icon, which is great, but there is no option to close the app window but keep it running in tray icon.
I hope this feature will be there.

**Comment by elelay** [comment]
Just click on the tray icon, right ?

**Comment by jalalsfs** [comment]
If I click on the tray icon it is working, but what I mean is that, if the app has an option when I click on close the app should be working and be in tray icon.

**Comment by elelay** [comment]
OK. I'll reopen
